src/core/qgismobileapp.cpp

```diff
@@ -573,8 +569,12 @@ void QgisMobileapp::initDeclarative( QgsApplication *mApp, QQmlEngine *engine )
   engine->rootContext()->setContextProperty( "platformUtilities", PlatformUtilities::instance() );
 }
 
-void QgisMobileapp::registerGlobalVariables()
+void QgisMobileapp::registerGlobalVariables( QgsApplication *mApp )
```

Comment:
We don't need the mApp there right? That's not a static function, it'll have access to its private members

Reply:
yes yes absolutely right, thanks. :+1: